src/WebJobs.Extensions.DurableTask/Listener/DurableTaskTriggerMetrics.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#if NETSTANDARD2_0
+using System;
+using System.Collections.Generic;
+using System.Text;
+using DurableTask.AzureStorage.Monitoring;
+using Microsoft.Azure.WebJobs.Host.Scale;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    internal class DurableTaskTriggerMetrics : ScaleMetrics
+    {
+        /// <summary>
+        /// The number of partitions in the task hub.
+        /// </summary>
+        public int PartitionCount { get; set; }
+
+        /// <summary>
+        /// The number of messages across control queues.
+        /// </summary>
+        public string ControlQueueLengths { get; set; }
+
+        /// <summary>
+        /// The latency of messages across control queues.
+        /// </summary>
+        public string ControlQueueLatencies { get; set; }
+
+        /// <summary>
+        /// The number of messages in the work-item queue.
+        /// </summary>
+        public int WorkItemQueueLength { get; set; }
+
+        /// <summary>
+        /// A trend describing the latency of messages in the work-item queue

```

Comment:
Unclear from the comments how this double matches a trend. Is it some fractional value between -1 and 1, is it an average of many integers, etc.

Reply:
This is actually left over from PerformanceHeartbeat, so I'm removing it -- it's not necessary for MakeScaleRecommendation